### PR TITLE
Show message for invites which are already part of org.

### PR DESCRIPTION
**Pull request by imabp**
We recently had this issue, [#156](https://github.com/Tech-Phantoms/community/issues/156) an existing user is requesting the GitHub invite. 
We want to display a message in the comment, that you are already part of the org.

image

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const main = async () => {
     const org = core.getInput('organization', { required: true });
     const label = core.getInput('label', { required: true });
     const comment = core.getInput('comment');
+    const alreadyAMember = `You are already the member of: ${org}`
```
Comment (vj-abishek) [inline index.js:22]: Looks good 

Instead of declaring variable here. Get it from the input. So that the value can be changed

Comment (vj-abishek) [reply index.js:22]: Add a new input here. And give it a default value

https://github.com/vj-abigo/invite-on-label/blob/main/action.yml

Comment (imabp) [reply index.js:22]: Sure I will raise a new PR, theres a glitch in matrix..on my locals

**Comment by vj-abishek** [comment]
Hello @imabp, still working on this?

**Comment by vj-abishek** [comment]
Hello @imabp, this issue is being solved in [v1.3](https://github.com/vj-abigo/invite-on-label/releases/tag/v1.3)

**Comment by imabp** [comment]
Awesome thanks, I am bit occupied with other work. Cannot Contribute. But thankful, we are using this at our open source org
[tech-phantoms/community](https://github.com/tech-phantoms/community)

**Pull request closed without merging**